src/k_meanspp.rs

```diff
@@ -0,0 +1,347 @@
+// http://rosettacode.org/wiki/K-means%2B%2B_clustering
```

Comment:
Can you run the `coverage` crate against this task? IIRC URL-encoding the comment confuses it.